torch/_torch_docs.py

```diff
@@ -3137,12 +3137,28 @@ def merge_dicts(*dicts):
 The dividend and divisor may contain both for integer and floating point
 numbers. The remainder has the same sign as the dividend :attr:`input`.
 
-When :attr:`other` is a tensor, the shapes of :attr:`input` and
-:attr:`other` must be :ref:`broadcastable <broadcasting-semantics>`.
+Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`,
+:ref:`type promotion <type-promotion-doc>`, and integer and float inputs.
+
+.. note::
+
+    When the divisor is zero, returns ``NaN`` for common dtype as
+    floating-point on both CPU and GPU; raises ``RuntimeError`` for
+    common dtype as integral on CPU;
+
+    For common dtype as integral On GPU, due to it's an undefined behavior,
```

Comment:
This is interesting but too detailed for this portion of the documentation (if we had an implementation details section we could put it there). Instead, I suggested removing this paragraph and adding a sentence to the previous paragraph:

"Integer division by zero on the GPU may return any value." 

I understand how the use of "common dtype" above is more precise, and it's good to be so precise, but unfortunately the typical reader of the documentation won't understand the concept. That's why I've suggested simplifying the language above.